src/api/get-profile-image.js

```diff
@@ -0,0 +1,20 @@
+const collectionIdsUrl = name => `https://collectionapi.metmuseum.org/public/collection/v1/search?q=${name}`;
+const imageUrl = id => `https://collectionapi.metmuseum.org/public/collection/v1/objects/${id}`;
+
+async function getData(url) {
+  const fetchBody = await fetch(url);
+  const json = await fetchBody.json();
+  return json;
+}
+
+function randomImg(imgIds) {
+  return imgIds[Math.floor(Math.random() * imgIds.length)];
+}
+
+export default async function (name) {
+  const collection = await getData(collectionIdsUrl(name));
+  //[291826] = 'unknown'. It is used here as a profile for any name that returns no images.
```

Comment:
Rather than commenting, make it a const
```suggestion
  const unknownName = 291826;
  const imageId = randomImg(collection.objectIDS || [unknownName])
```